<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="local">
       <attribute name="title">
        <string>Local</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Available chapters for edit:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="editor_menu"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="edit_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Edit Chapter</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="return_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Return Previous</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="show_versions_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Show Versions</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="show_changes_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Show Changes</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="reload_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Reload Chapter</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="push_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Push to server</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="server">
       <attribute name="title">
        <string>Server</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTreeWidget" name="books_tree">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <column>
           <property name="text">
            <string>Books</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>User Position</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Book Status</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttons_layout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMaximumSize</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="user_btns_layout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QPushButton" name="add">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Add Chapter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clone">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Clone Chapter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="export_html">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Export HTML</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>30</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="admin_btns_layout">
            <item>
             <widget class="QPushButton" name="add_author_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Add Author</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="remove_author_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Remove Author</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="change_name_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Change Name</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="publish_btn">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Publish</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_Book">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Book"/>
    <addaction name="separator"/>
    <addaction name="actionLog_Out"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Us"/>
   </widget>
   <addaction name="menuAdd_Book"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout_Us">
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
  <action name="actionNew_Book">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New Book</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
